_posts/en/posts/2018-09-20-notice.md

```diff
@@ -60,6 +60,8 @@ Timeline for September 17, 2018: (all times UTC)
 September 18, 2018:
 
 - 00:24 Bitcoin Core version 0.16.3 tagged
+- 14:54 Bitcoin Knots version 0.16.3.knots20180918 tagged
+- 19:23 Bitcoin Knots release binaries and release announcements were available
```

Comment:
This is a website for Bitcoin Core, I don't think this information is necessary.

Reply:
Agree on this

Reply:
Then we should remove ABC too?